### DISCUSSION TITLE
Item Predicates for aiming mode

### DISCUSSION THREAD

**Discussion by PixelatedFont** [Leawind/Third-Person#184] [Q&A] [answered]
Either I'm blind or I can't find documentation for this, so through trials and errors, I figured it out that (ignore the quotations) 'stick' 'arrow' 'golden_hoe' works. 

So, are there any more rules? Is it possible to make it so that for as long as a particular word is in the name of the item i.e: 'Bow of', so that I can make a bunch of items named "Bow of Fire", "Bow of Knockback" and it would work with only one line of rule?

I am running a lots of mod so if it already can do that and that my mods setup is messing it up do let me know.

Many thanks, for reading this and making this mod.

**Comment by Leawind** [comment, marked as answer]
You mean by the translated name of item? Currectly there is no such rule.

The full documentation is here: https://leawind.github.io/Third-Person/en-US/Details/ItemPredicate
